#!/Users/imac/Desktop/Data_Scince/env/bin/python3.9
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
